nekoyume/Assets/_Scripts/UI/BattleResult.cs

```diff
@@ -301,10 +302,10 @@ private void UpdateView(bool isBoosted)
         private IEnumerator CoUpdateViewAsVictory(bool isBoosted)
         {
             AudioController.instance.PlayMusic(AudioController.MusicCode.Win, 0.3f);
-            StartCoroutine(EmitBattleWinVFX(isBoosted));
+            StartCoroutine(EmitBattleWinVFX());
 
             victoryImageContainer.SetActive(true);
-            _victoryImageAnimator.SetInteger("ClearedWave",
+            _victoryImageAnimator.SetInteger(ClearedWave,
                 isBoosted ? 4 : SharedModel.ClearedWaveNumber);
```

Comment:
How about make this constant into `const` variable commenting what this line means?